docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+categorie;element;leveren door RvIG;Beperkt zoekresultaat;Persoon GBA variant;;;;;opmerking/issue;
```

Comment:
zou je ook de kolom Buitengemeentelijke autorisatie kunnen toevoegen?

Reply:
Heb ik gedaan.

Wat betekent trouwens de waarde '~'precies? Moet ik wellicht ook verklaren in 'mapping.md'.